lldb/source/Plugins/SymbolVendor/ELF/SymbolVendorELF.cpp

```diff
@@ -44,6 +44,25 @@ llvm::StringRef SymbolVendorELF::GetPluginDescriptionStatic() {
          "executables.";
 }
 
+// If this is needed elsewhere, it can be exported/moved.
+static bool IsDwpSymbolFile(const lldb::ModuleSP &module_sp,
+                            const FileSpec &file_spec) {
+  DataBufferSP dwp_file_data_sp;
+  lldb::offset_t dwp_file_data_offset = 0;
+  // Try to create an ObjectFile from the file_spec.
+  ObjectFileSP dwp_obj_file = ObjectFile::FindPlugin(
+      module_sp, &file_spec, 0, FileSystem::Instance().GetByteSize(file_spec),
+      dwp_file_data_sp, dwp_file_data_offset);
+  if (!ObjectFileELF::classof(dwp_obj_file.get()))
```

Comment:
You probably want to do 
`if (!dwp_obj_file && !ObjectFileELF::classof(dwp_obj_file.get()))`
dwp_obj_file might be a null pointer

Reply:
Fixed in PR https://github.com/llvm/llvm-project/pull/86292 Good catch, @GeorgeHuyubo !